src/ActivityLogger.php

```diff
@@ -2,129 +2,123 @@
 
 namespace Spatie\Activitylog;
 
-use Illuminate\Auth\AuthManager;
 use Illuminate\Contracts\Config\Repository;
 use Illuminate\Database\Eloquent\Model;
 use Illuminate\Support\Arr;
 use Illuminate\Support\Carbon;
 use Illuminate\Support\Str;
 use Illuminate\Support\Traits\Macroable;
 use Spatie\Activitylog\Contracts\Activity as ActivityContract;
-use Spatie\Activitylog\Exceptions\CouldNotLogActivity;
 
 class ActivityLogger
 {
     use Macroable;
 
-    /** @var \Illuminate\Auth\AuthManager */
-    protected $auth;
+    protected string $defaultLogName = '';
```

Comment:
I think that we should use the propper "nothing" value PHP provides here.

```suggestion
    protected ?string $defaultLogName = null;
```